### ISSUE TITLE
Can't install gplaycli with Python 2.7.12

### ISSUE THREAD

**Issue by tsekityam** [matlink/gplaycli#127]
I WON'T ANSWER IF YOU DON'T PROVIDE SUCH DETAILS. COPY-PASTING IS NOT ENOUGH, I'M NOT A JEDI (YET).

Please provide those informations:

- [x] Operating System (ie Ubuntu 16.04)
```
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 16.04.3 LTS
Release:        16.04
Codename:       xenial
```

- [x] Python version when running `gplaycli` (should be 2.7 for now)
```
Python 2.7.12
```

- [ ] GPlayCli version via `gplaycli -v`
N/A

- [ ] The way you installed `gplaycli`: `git clone`, `git clone and setup.py`, `pip install gplaycli`, `apt install gplaycli`, ...
```
$ pip install gplaycli
```

- [ ] The authentication method: (own) credentials or (own) token.
N/A

******
I got the following errors when I tried to install the program via `pip`.
```
Collecting gplaycli
  Using cached GPlayCli-3.10.tar.gz
    Complete output from command python setup.py egg_info:

        Python2 support has been removed since version 3.9
        Please install GPlayCli with Python3
        Or install version 3.8 but don't expect support

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-gAHlyy/gplaycli/
```

Perhaps it is a duplication of #124.

FYI, here is the terminal log of the installation
```
tsekityam@EELNB1067:~$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 16.04.3 LTS
Release:        16.04
Codename:       xenial
tsekityam@EELNB1067:~$ python --version
Python 2.7.12
tsekityam@EELNB1067:~$ sudo apt-get install python-dev
Reading package lists... Done
Building dependency tree
Reading state information... Done
python-dev is already the newest version (2.7.11-1).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
tsekityam@EELNB1067:~$ sudo apt-get install libffi-dev
Reading package lists... Done
Building dependency tree
Reading state information... Done
libffi-dev is already the newest version (3.2.1-4).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
tsekityam@EELNB1067:~$ sudo apt-get install libssl-dev
Reading package lists... Done
Building dependency tree
Reading state information... Done
libssl-dev is already the newest version (1.0.2g-1ubuntu4.8).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
tsekityam@EELNB1067:~$ pip install gplaycli
Collecting gplaycli
  Using cached GPlayCli-3.10.tar.gz
    Complete output from command python setup.py egg_info:

        Python2 support has been removed since version 3.9
        Please install GPlayCli with Python3
        Or install version 3.8 but don't expect support

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-gAHlyy/gplaycli/
```

**Comment by tsekityam** [comment]
The README said that the min. requirement is Python 2.7, but the requirement is changed to Python3. I think we should update the README to reflect the changes in `GPlayCli-3.9`.

**Comment by abhijeet0007** [comment]
you have to use the python3, install python3
check python3 version (python -V )
then you have to install pip3
then you have to command ( use pip3 install gplaycli )

**Comment by matlink** [comment]
Yes you're right I totally forgot to change the readme requirements.

**Comment by matlink** [comment]
Fixed in 6e9d84b8fbabeaeda714d72c2002eb7d16ac3b5d
